example2.rb

```diff
@@ -0,0 +1,9 @@
+class Example2
+
+  def one
+    p 'one'
+  end
+  def two
+    p 'two'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.